distribution/ddf-common/src/main/resources/etc/users.properties

```diff
@@ -33,4 +33,4 @@
 # DDF Changes: changed username/pw to admin/admin, added localhost certificate user
 
 admin=admin,group,admin,manager,viewer,webconsole,system-admin,systembundles
-localhost=localhost,group,admin,manager,viewer,webconsole,system-admin,systembundles
\ No newline at end of file
+localhost=localhost,group,admin,manager,viewer,webconsole,system-admin,systembundles,codice-history
```

Comment:
Looks like we went with system-* for DDF specific roles.   We should probably rename the history role to system-history.